assets/js/main.js

```diff
@@ -156,6 +156,32 @@ $(document).ready((function ($) {
         return false;
     });
 
+    $('.appointment-button').click(function () {
+        var $form = $('.appointment-form'),
+            data = getFormData($form),
+            host = 'http://127.0.0.1:8000/',
+            path = 'api/appointment/',
+            url = host + path;
+
+        $.ajax({
+            type: 'POST',
+            url: url,
+            data: JSON.stringify(data),
+            beforeSend: removeFieldErrors,
+            success: function () {
```

Comment:
I can't successfully create an appointment because serializer fields in the backend do not match the name attributes used in the frontend. I see you have fixed that in the barbers feature branch, but please make sure you finish one feature before you move on to the next one. 

I'll continue with the review once you'll set up the appointment feature properly in `feature/appointment` branch. Move all the necessary parts for the appointment feature from `feature/barbers` branch here.